Makefile

```diff
@@ -35,7 +35,7 @@ test_fast::
 	go test -timeout $(TEST_FAST_TIMEOUT) -count=1 -parallel ${TESTPARALLELISM} ${PROJECT_PKGS}
```

Comment:
this probably needs a .Short() as well.

Reply:
A good idea  not sure it ends up mattering in practice, given how we have stuff set up today.